### DISCUSSION TITLE
axios returns 406 on redirect

### DISCUSSION THREAD

**Discussion by ningji** [axios/axios#4542] [Q&A] [answered]
My node.js server code is like this

const doRedirect = (req, res) {
    return res.redirect('some url here');
}

app.post('/some_path', doRedirect);
in my test,

const loginOptions = (body, cookie) => {
    return {
        url: 'http://localhost/some_path',
        data: body,
        method: 'POST',
        headers: {
            Cookie: cookie,
            'User-Agent': getUserAgent()
        },
    };
};

it('my redirect test', async () => {
    const some_tests = loginOptions({}, '123'); 
    try {
        const res = await axios(some_request);
    } catch (e) {
        console.error('why hit here ?');
    }
}
When i test redirect manually in local node server, it does the redirect just fine.

But to my surprise, the res.status is 406, why is this ?

**Comment by carpben** [comment, marked as answer]
I think the proper place for such questions is Stackoverflow (Or profressional forums), who are build for it, and where you are more likely to get a reply. As I see it, this discussion board is more intended for discussions related to the library itself. 
I suggest closing it here and opening it somewhere else. 
@jasonsaayman let me know if you think otherwise.
